homeassistant/components/benq_projector/README.md

```diff
@@ -0,0 +1,25 @@
+# Serial-BenQ-Projector-Home-Assistant-Integration
```

Comment:
Please remove the README file, this should be part of our documentation instead.